From 4b82d986f9fbc13827cc8f6d771efd0adbaab880 Mon Sep 17 00:00:00 2001
From: Jun Wu <jun.wu@arm.com>
Date: Fri, 26 Jul 2024 09:35:46 +0000
Subject: [PATCH] check-sr-results/dt-parser: Update to Linux 6.10 for
 arm,mhuv3.yaml

Update Linux Kernel version to 6.10 to include the dt-binding for
arm,mhuv3, remove arm,mhuv3 in extra_compat of check-sr-results.yaml
and ignored rule in dt-parser.yaml.

Upstream-Status: Inappropriate [Kronos-specific update]

Signed-off-by: Jun Wu <jun.wu@arm.com>
---
 check-sr-results.py   | 2 +-
 check-sr-results.yaml | 1 -
 dt-parser.yaml        | 8 --------
 3 files changed, 1 insertion(+), 10 deletions(-)

diff --git a/check-sr-results.py b/check-sr-results.py
index cd5d788cd0e9..a7ee8b6979f0 100755
--- a/check-sr-results.py
+++ b/check-sr-results.py
@@ -1612,7 +1612,7 @@ if __name__ == '__main__':
     parser.add_argument(
         '--linux-url', help='Specify Linux tarball URL',
         default='https://cdn.kernel.org/pub/linux/kernel/v6.x/'
-                'linux-6.3.7.tar.xz')
+                'linux-6.10.tar.xz')
     parser.add_argument(
         '--parser', help='Specify (SCT) parser.py path',
         default='parser.py')
diff --git a/check-sr-results.yaml b/check-sr-results.yaml
index e6b37e690ddb..c73d862ff524 100644
--- a/check-sr-results.yaml
+++ b/check-sr-results.yaml
@@ -339,7 +339,6 @@ tree:
               - arm,neoverse-v3ae
               - arm,slc
               - arm,mpam-msc
-              - arm,mhuv3
               - arm,si-rproc
               - arm,si-channel
           - file: BsaDevTree.dtb.log  # This will be generated
diff --git a/dt-parser.yaml b/dt-parser.yaml
index 34bdd7d3b04a..f5fb06c20d2e 100644
--- a/dt-parser.yaml
+++ b/dt-parser.yaml
@@ -214,14 +214,6 @@
   update:
     type: ignored
 
-- rule: Allow no schema for arm,mhuv3
-  criteria:
-    type: dt-validate warning
-    warning_message: 'failed to match any schema with compatible: '
-    in_compatibles: 'arm,mhuv3'
-  update:
-    type: ignored
-
 - rule: Allow no schema for arm,rd1-ae
   criteria:
     type: dt-validate warning
-- 
2.34.1

